AK/Format.cpp

```diff
@@ -374,6 +374,53 @@ void FormatBuilder::put_f64(
 
     put_string(string_builder.string_view(), align, min_width, NumericLimits<size_t>::max(), fill);
 }
+
+void FormatBuilder::put_f80(
+    long double value,
+    u8 base,
+    bool upper_case,
+    Align align,
+    size_t min_width,
+    size_t precision,
+    char fill,
+    SignMode sign_mode)
```

Comment:
This is just `put_f64` with some types changed to `long double`. Is there a better way to do this that doesn't require copying code?

Reply:
I can make it a
```cpp
template<FloatingPoint T>
void FormatBuilder::put_float(
    T value,
    ...
)
{
    ...
}
```
instead, that might be a better idea even

Reply:
GCC does not seem very cooperative today,
If you get it working, just make a pull request to my repo and I will merge it then